### PR TITLE
Update site to new design

### PR DESCRIPTION
**Pull request by nijynot**
Updates the site to the new design.
Live example: https://nijynot.github.io/site

### CODE REVIEW

**Comment by arad54** [comment]
The "Grin - The Mimblewimble Blockhchain" header and the content beneath is quite hard to read.
May I suggest switching the font used there altogether and making the header less dense; bigger, thinner, more space between characters.
Also tr to make the sub-headers more distinguishable. Those paragraphs feel like a bit white blur to me.

Great job overall!

**Comment by 0xb100d** [comment]
- I miss the old yellow hand-drawn icons for private scalable and open... though I can see how they might not fit on the new minimal design.

- very trivial but if the grin logo in the top right spun the other way scrolling down it makes the smile look like a leading arrow a bit more (I think it looks better scrolling up on the page then down as it is)

- community link now just links to community projects, but I think it should be a drop down including not only projects but the forum, the gitter, etc as the current site does.

- Wiki/FAQ should be more prominent, maybe another header menu link

- I think the Friends of Grin and the General Fund could be on the same drop down menu labelled Funding or something (I love the look of both pages though, but don't think they each need a main menu space).

-at the bottom it links to the old domain, now it is grin.mw

-maybe instead of "*" asterisk as the spacer between the text introduction and the next image use the international currency symbol ¤ [https://en.wikipedia.org/wiki/Currency_sign_(typography)]

it looks really good

**Comment by nijynot** [comment]
@arad54 Agree with you, the text is a bit hard to parse from first glance. Takes some time to understand it with the new structure and without the yellow hand-drawn icons which @0xb100d mentioned. Maybe we should add some new icons?

@0xb100d
> very trivial but if the grin logo in the top right spun the other way scrolling down it makes the smile look like a leading arrow a bit more (I think it looks better scrolling up on the page then down as it is)

A bit on the fence on this one. I think a clockwise rotation makes sense when scrolling down, while a counter-clockwise one would feel a bit unintutitive. I do like the thought of the mouth spinning more like an arrow though!

> Wiki/FAQ should be more prominent, maybe another header menu link

Yeah, agreed, should be there. Thought of setting it as `Resources`, a page that links to the wiki and other content.

> I think the Friends of Grin and the General Fund could be on the same drop down menu labelled Funding or something (I love the look of both pages though, but don't think they each need a main menu space).

Ah, I like this! Had some problems with displaying all the pages in a simple way. Introducing dropdowns again is probably the best way manage this.

> maybe instead of "*" asterisk as the spacer between the text introduction and the next image use the international currency symbol ¤ [https://en.wikipedia.org/wiki/Currency_sign_(typography)]

Hmm, while I do think it makes sense to add this instead of an asterisk, I don't think it looks as good. Don't have a big opinion on this one, so willing to change if anyone wants the ¤.

**Comment by arad54** [comment]
Another way you could go about making the content easier to read, besides adding icons which hardly fit into this design, is to minimize the sub-headers down to a single word like in the previous website (or two words max) and then break a line. 

Additionaly, change their font and make em’ distinguisheble by coloring the text white and creating an in-line black color-background. That’s just one way to go about it that popped off in my mind. I would demonstrate but I’m on my phone.

**Comment by lehnberg** [comment]
Merging this now, but this doesn't mean that the work is finished! Continue the feedback and discussion on the tracking issue: https://github.com/mimblewimble/site/issues/148
